qkit/drivers/Spectrum_M4i2211.py

```diff
@@ -164,43 +165,66 @@ def _load_dll(self):
         '''
         if platform.architecture()[0] == '64bit':
             pf_64Bit = True
+            drv_handle= POINTER(c_uint64)
         else:
             pf_64Bit = False
+            drv_handle = c_void_p
 
         if pf_64Bit:
             logging.debug(__name__ + ' : Loading spcm_win64.dll')
             self._spcm_win32 = windll.LoadLibrary('C:\\WINDOWS\\System32\\spcm_win64.dll')
 
             #function names are not decorated in the 64 bit version
-            self._spcm_win32.open           = self._spcm_win32["spcm_hOpen"]
-            self._spcm_win32.close          = self._spcm_win32["spcm_vClose"]
-            self._spcm_win32.SetParam32     = self._spcm_win32["spcm_dwSetParam_i32"]
-            self._spcm_win32.SetParam64m    = self._spcm_win32["spcm_dwSetParam_i64m"]
-            self._spcm_win32.SetParam64     = self._spcm_win32["spcm_dwSetParam_i64"]
-            self._spcm_win32.GetParam32     = self._spcm_win32["spcm_dwGetParam_i32"]
-            self._spcm_win32.GetParam64m    = self._spcm_win32["spcm_dwGetParam_i64m"]
-            self._spcm_win32.GetParam64     = self._spcm_win32["spcm_dwGetParam_i64"]
-            self._spcm_win32.DefTransfer64m = self._spcm_win32["spcm_dwDefTransfer_i64m"]
-            self._spcm_win32.DefTransfer64  = self._spcm_win32["spcm_dwDefTransfer_i64"]
-            self._spcm_win32.InValidateBuf  = self._spcm_win32["spcm_dwInvalidateBuf"]
-            self._spcm_win32.GetErrorInfo   = self._spcm_win32["spcm_dwGetErrorInfo_i32"]
+            self._spcm_win32.open           = getattr (self._spcm_win32, "spcm_hOpen")
```

Comment:
Is there a specific reason why you changed from `self._spcm_win32[attr]` to  `getattr(self._spcm_win32,attr)`? I was thinking these expressions are equal

Reply:
Here we were looking at python 3 scripts the manufacturers of the card had as examples and since, at the beginning, we did not know what were the problems with python 3, we used their definitions.
I also think they are exactly the same definition.